src/main/lang/de.json

```diff
@@ -218,6 +218,7 @@
         "experience": "Erfahrung"
       },
       "magic": {
+        "potency": "Wirksamkeit",
```

Comment:
How about "Potenz" instead?

Reply:
_Maybe_. I don't think it has quite the same connotations and meaning, and it frankly sounds a bit odd. What about "Wirkstärke"?

Reply:
Though "Potenz" is the direct equivalent. Yes in some heads this will raise a different connotation, but I think it fits.